Packs/CommonPlaybooks/Playbooks/playbook-User_Investigation_-_SIEM_search_for_failed_logon_-_Generic.yml

```diff
@@ -0,0 +1,706 @@
+id: User Investigation - SIEM search for failed logon - Generic
+version: -1
+name: User Investigation - SIEM search for failed logon - Generic
```

Comment:
Should change the name (from my readme comment)

Reply:
Done.